fr/core-libraries/collections.rst

```diff
@@ -151,18 +152,19 @@ dot notation paths::
     ];
     $combined = (new Collection($items))->combine('id', 'name');
 
-    // Result will look like this when converted to array
+    // Le résultat ressemblera à ceci quand il est converti en tableau
```

Comment:
I have a doubt on this _(fr : niveau concordance des temps)_. 
I'll either put
_Le résultat ressemblera à ceci s'il est converti en tableau_
ou
_Le résultat ressemblera à ceci quand il sera converti en tableau_

I'll let you chose either or keep the original one.